docs/java-run-your-first-app.md

```diff
@@ -0,0 +1,160 @@
+By running the application as is, at minimum you got a taste of one of Temporal's amazing value propositions: visibility into the Workflow and the status of the Workers executing the code. Let's explore another key value proposition, maintaining the state of a Workflow, by simulating some failures. Make sure your Worker is stopped before proceeding.
+
+### Server crash
+
+In most modern application environments, to accommodate a host failure, leader election processes are applied to multiple application instances which are backed by databases that maintain the application state and so on. With Temporal, even if the Temporal server is down, the state of the Workflow is preserved. You can easily test this by following these steps (again, make sure your Worker is stopped so your Workflow doesn't finish):
+
+1. Start the Workflow again.
+2. Verify the Workflow is running in the UI.
+3. Shut down the Temporal server by either using 'Ctrl c' or via the Docker dashboard.
+4. After it has stopped, restart the Temporal server and visit the UI.
+
+The Workflow is still there!
+
+### Activity error
+
+Next let's simulate a bug in the Activity method. Inside your project find the AccountActivityImpl.java file and uncomment the line that throws an exception in the deposit method:
+
+<!--SNIPSTART money-transfer-project-template-java-activity-implementation-->
+<!--SNIPEND-->
+
+Save it and run the Worker (the Workflow should still be running, start it if it isn't). The Worker executes the Workflow, and completes the `withdraw()` Activity. But then the exception is thrown when it attempts the `deposit()` Activity. Notice how the Worker keeps retrying the Activity? Visit the [UI](localhost:8088) and click on the RunId of the Workflow. You will see the pending Activity listed there with details such as its state, the number of times it has been attempted, and the next scheduled attempt.
+
+![Activity UI error details](/img/docs/web-ui-activity-error-info.png)
+
+[Timeout configurations](/docs/activities/#timeouts) and [retry policies](/docs/activities/#retries) for Activities are specified in the Workflow code as Activity options. This is another key value proposition that Temporal offers, as other modern applications implement timeout and retry logic within the business process itself. In our Workflow code you can see that we have specified a setStartToCloseTimeout for our Activities, and set a retry policy that tells the server to retry them up to 500 times. But we did that as an example for this tutorial, as Temporal automatically uses a default retry policy if one isn't specified!
+
+So, your Workflow is running, but only the `withdraw()` Activity succeeded. In any other application, the whole process would likely have to be abandoned and rolled back. But with Temporal, we can test the durable state of the Workflow again by stopping either the Temporal server, the Worker, or both and then starting them back up again. The Workflow will be in the same state!
```

Comment:
```suggestion
So, your Workflow is running, but only the `withdraw()` Activity method succeeded. If you weren't using Temporal the whole process would likely have to be abandoned and rolled back. But with Temporal, we can levrage the durable state of the Workflow by stopping either the Temporal server, the Worker, or both and then starting them back up again. The Workflow will be in the same state!
```

"But with Temporal, we can levrage the durable state of the Workflow by stopping either the Temporal server, the Worker, or both and then starting them back up again. The Workflow will be in the same state!"

That's the same value as the other time you talk about it. This needs to incorporate the live debugging and don't mention the server.